vuepress/docs/next/tutorials/solution/entando-hub.md

```diff
@@ -65,24 +64,77 @@ ent ecr deploy --repo=docker://registry.hub.docker.com/entando/entando-hub-conte
    - Give at least one user the ability to manage the hub by granting the `eh-admin` role. Assign the `eh-admin` role for the `pn-152edaba-0a2ba8fb-entando-entando-hub-catalog-ms-server` client. See [Role Assignment in ID Management](../../docs/consume/identity-management.md#authorization) for more details.
    - Give the generated plugin client permission to manage users. 
        1. From the left sidebar, go to `Clients` and select client ID `pn-152edaba-0a2ba8fb-entando-entando-hub-catalog-ms-server`. 
        2. Click the `Service Account` tab at the top of the page and select `realm-management` from the `Client Roles` field. 
        3. Choose `realm-admin` from `Available Roles` and click `Add selected`. It should now appear as an `Assigned Role`.
     
-2. Access your enterprise hub from the App Builder by navigating to `Pages → Management`. Find `Entando Hub` in the page tree, and click `View Published Page` from its actions.
-
-::: tip
-(Entando 7.0+) Any enterprise hub instance can be accessed from the Entando App Builder of another Entando Application. Configure the App Builder to access the desired hub instance via the endpoint `BASEURL/entando-hub-api/appbuilder/api`, where the BASEURL is the URL for the Entando Application.
-:::
+2. Access your enterprise hub from the App Builder by navigating to `Pages → Management`. Find `Entando Hub` in the page tree, and click `View Published Page` from its Actions.
 
 ## Using an Entando Enterprise Hub
+### The Hub UI
+The Entando enterprise Hub is equipped with a user interface from where users, entries, and catalogs are managed. Private and public catalogs can also be configured here. 
+* Administrator can create and manage users, categories, and organizations. 
+* Authors and managers have varying [levels of access](#roles) to create and manage entries, otherwise called bundle groups.
+* Each catalog can be [connected directly to an App Builder](#add-a-catalog-as-a-registry-in-your-app-builder) instance for easy access.
+
+![Hub UI](./hub-images/hub-ui.png)
+
+### User Management
+Only the Hub admin has the authorization to create and manage users. 
+  1. Log into your Keycloak admin console 
+  2. Go to the `Users` section from the left navigation bar and add a new user. Enter the relevant identity information. 
+  3. Once saved, go to the `Role Mapping` tab and assign the correct role under `Client Role` `pn-152edaba-0a2ba8fb-entando-entando-hub-catalog-ms-server`
+    * for an author, assign `eh-author`
+    * for a manager, assign `eh-manager`
+  4. Log in to the Hub UI as an admin 
+  5. Go to `User Management` and click `Add User`
+  6. Choose the desired user and select an organization from the drop-down list. If the organization is not available, go to Organization Management to add it.
+  7. Note that an admin user needs to belong to an organization as well, especially for private catalogs that require an API key. 
```

Comment:
Note that a user, including the admin user, needs to belong to an organization to generate API keys required for private catalogs